pytorch_blade/bazel_build.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, *args, **kwargs):
             "@org_tensorflow//tensorflow/compiler/mlir/xla/ral:libral_base_context.so",
             "//src:libtorch_blade.so",
             "//src:_torch_blade.so",
+            "//tests/mhlo/torch-mlir-opt:torch-mlir-opt",
```

Comment:
Nit: why is `torch-mlir-opt` built here? It's only used by file checks

Reply:
It's easy to debug some mlir files without running any filecheck uts. And it's useful to test building `torch-mlir-opt`.